common/src/warp_runner/manager/commands/raygun_commands.rs

```diff
@@ -106,8 +116,29 @@ pub async fn handle_raygun_cmd(
             };
             let _ = rsp.send(r);
         }
-        RayGunCmd::SendMessage { conv_id, msg, rsp } => {
-            let r = messaging.send(conv_id, None, msg).await;
+        RayGunCmd::SendMessage {
+            conv_id,
+            msg,
+            attachments,
+            rsp,
+        } => {
+            let r = if attachments.is_empty() {
+                messaging.send(conv_id, None, msg).await
+            } else {
+                messaging.attach(conv_id, attachments, msg).await
```

Comment:
Just as a quick note, since I just noticed it, messages here in the attach function are optional. So down the road, might want to lift it up so it could send an attachment even if a message is not present for `Raygun::attach`. Doesnt block this PR though unless its something you wish to quickly do.